runtime/internal/voice.js

```diff
@@ -182,9 +182,20 @@ function next (msg, suffix, bot) {
       if (connection.voiceConnection.guild.id === msg.guild.id) {
         if (list[msg.guild.id].link.length === 0) {
           delete list[msg.guild.id]
```

Comment:
Not sure if you tested this but this delete should only run when the bot has left voice not if it's staying.